3rdparty/python/requirements.txt

```diff
@@ -3,6 +3,10 @@ beautifulsoup4>=4.6.0,<4.7
 fasteners==0.15.0
 freezegun==1.0.0
 
+# Note: we use humbug to report telemetry. When upgrading, ensure the new version maintains the
+# anonymity promise we make here: https://www.pantsbuild.org/docs/anonymous-telemetry
```

Comment:
I think this page is missing answers to a couple of important questions:
* Who can access the data, and with whom it might be shared
* How long the data will be retained

Reply:
Yeah will add some GDPR info to cover this.

Reply:
Done

Reply:
How will we provide access or deletion given the data is anonymised?

Reply:
You provide us with a repo id and we sha256 it. If the repo was private then only you have access to your own repo id. If the repo was public we can verify your ownership of it.